assets/js/components/DashboardNavigation.js

```diff
@@ -142,7 +154,7 @@ export default function DashboardNavigation() {
 		}
 
 		const chipID = hash.substring( 1 );
-		setIsJumpingTo( chipID );
+		setValue( UI_VIEW_CONTEXT, chipID );
```

Comment:
A minor detail - the IB mentions to replace the use of `hash` and `chipID` in this function with `isJumpingTo`, as `isJumpingTo` will contain the value `global.location.hash.substring(1)` at this point.

However, this is a non-functional change and if you find it clearer as it stands, that's fine by me!